docs/3.0rc/resources/upgrade-prefect-3.mdx

```diff
@@ -0,0 +1,207 @@
+---
+title: Upgrade to Prefect 3
+description: Learn how to upgrade from Prefect 2 to Prefect 3.
+---
+
+Some patterns supported by Prefect 2 are no longer supported with Prefect 3, or are enabled differently. 
+This guide covers changes that you may need to make to ensure your flows keep flowing.
+
+## Module locations and names
+
+Some modules have been renamed, reorganized, or removed for clarity. If you use a module that has changed, it will continue to work, but it will raise deprecation warnings. You will need to update these modules before the six month deprecation period is up:
```

Comment:
I believe this will be dependent on everyone ensuring they have added any moved or removed modules to the migration utility dictionary

Reply:
Yes, that's right. If we discover things that we missed. We'll have to add them both here and there.

Reply:
```suggestion
Some modules have been renamed, reorganized, or removed for clarity. If you use a module that has changed, it will continue to work, but it will raise deprecation warnings. You will need to update these modules before the six month deprecation period ends:
```